packages/fxa-react/styles/tailwind.css

```diff
@@ -6,4 +6,14 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer utilities {
+  @media (prefers-reduced-motion: reduce) {
+    svg,
+    svg * {
+      animation: none !important;
```

Comment:
Normally I really dislike `!important`, but this seems like a good use for it.

Reply:
Same, but felt it was ⭐ important ⭐ to respect the accessibility preference - plus I targeted only the SVGs. Maybe we'll eventually want to consider if we have other animations that should respect the reduced motion preference, but I figured that we have very few and most are short/non-repeating.